explorer/src/pages/Attestations/table/dataTable.tsx

```diff
@@ -0,0 +1,50 @@
+import { flexRender, getCoreRowModel, useReactTable } from "@tanstack/react-table";
```

Comment:
is there any point in this component if it already exists https://github.com/Consensys/linea-attestation-registry/blob/dev/explorer/src/components/DataTable/index.tsx